test/feature/support/helpers.coffee

```diff
@@ -61,8 +61,15 @@ class Helpers
     child.on 'message', (message) ->
       done() if message is 'server started'
 
+    exitSuccessful = no
```

Comment:
How about `unexpectedExit` with a default of yes and set it to no when you kill it?